var/spack/repos/builtin/packages/intel-oneapi-compilers/package.py

```diff
@@ -334,6 +350,12 @@ def install(self, spec, prefix):
         ftn = find("fortran-installer", "*")
         if ftn:
             self.install_component(ftn[0])
+        # install nvidia-plugin
+        if self.spec.satisfies("+nvidia"):
+            nvidia_script = find("nvidia-plugin-installer", "*")
+            if nvidia_script:
+                self.install_component(nvidia_script[0],is_nvidia=True)
+        
 
             # Some installers have a bug and do not return an error code when failing
             if not is_exe(self._llvm_bin.ifx):
```

Comment:
this code checks if the fortran compiler install succeded so should be immediately after 352.